### ISSUE TITLE
Order of gestureHandling and cluster icons (z-index)

### ISSUE THREAD

**Issue by octojoerg** [gmaps-marker-clusterer/gmaps-marker-clusterer#34]
I found an usability issue due to order of `gestureHandling` and cluster icons. 

By dragging the map with only one finger you receive a message á la **"Move the map with two fingers"** (in german "Verschieben der Karte mit zwei Fingern"). As you can see, the order of this hint message and cluster icons is not correct - cluster icons are on the top, but I think the hint message should be on the top.

However, it's possible to style the hint message with CSS class `.gm-style-pbt`, but no z-index property can fix this problem.

Any ideas?

![screenshot_2017-04-28-09-02-49](https://cloud.githubusercontent.com/assets/19168327/25518892/e9e0e950-2bf5-11e7-94e4-1d966fb768b2.png)

**Comment by Remo** [comment]
can you try to add `.gm-style-pbc { z-index: 9999 }`?

**Comment by Remo** [comment]
Update: While I can change the z-index, it breaks my clusters!

**Comment by octojoerg** [comment]
Yep, after changing z-index with all clusters are not selctable, also info-boxes can not be closed... :(

**Comment by Remo** [comment]
This is a tricky one, but here's something that might work, but make sure you are wearing your protective glasses, it's a tad bit ugly:
```css
 .gm-style-pbc[style*="opacity: 1"] { z-index: 9999 ! important }
```
I can't think of a more elegant way right now, but I'll keep this on my list, hopefully someone smarter sees this and comes up with a better idea.

**Comment by octojoerg** [comment]
Thanks, this work.

But sadly still hacky because of `!important`

**Comment by Remo** [comment]
Well, I find the `style*=` even hackier 😃 
I'll keep this open, I'd like to find a better solution for this, just can't think of anything right now.

**Comment by outaTiME** [comment]
Same here but using MarkerWithLabel (https://github.com/googlemaps/v3-utility-library/tree/master/markerwithlabel) i solve using hack from https://stackoverflow.com/a/44550389/460939 but the guys from google may be solve them

```
.gm-style-pbc {
  // show over markers
  z-index: 5 !important;
  pointer-events: none;
}
```

**Comment by outaTiME** [comment]
btw, why not report the error in the google repository? It would be very handy

**Comment by Remo** [comment]
@outaTiME the google repo isn't maintained anymore! I think I like the hack from stackoverflow a bit better than mine, but still not that much ;-)

**Comment by J4cku** [comment]
I actually prefer the one with "hacky" addressing than pointer-events, because of compatibility. Since there will be users that wont be able to click on markers when using old browsers
